packages/flutter/test/material/popup_menu_theme_test.dart

```diff
@@ -0,0 +1,295 @@
+                      PopupMenuItem<void>(
+                        key: popupItemKey,
+                        textStyle: textStyle,
+                        child: const Text(''),
+                      ),
+                ];
+                return list;
+                },
+            ),
+          ],
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byKey(popupButtonKey));
+    await tester.pumpAndSettle();
+
+    final Material button = tester.widget<Material>(
+      find.descendant(
+        of: find.byKey(popupButtonApp),
+        matching: find.byType(Material),
+      ).last,
+    );
+    expect(button.color, color);
+    expect(button.shape, shape);
+    expect(button.elevation, elevation);
+
+    final AnimatedDefaultTextStyle text = tester.widget<AnimatedDefaultTextStyle>(
+      find.descendant(
+        of: find.byKey(popupItemKey),
+        matching: find.byType(AnimatedDefaultTextStyle),
+      ).last,
+    );
+    expect(text.style, textStyle);
+  });
+
+  testWidgets('ThemeData.popupMenuTheme properties are utilized', (
+      WidgetTester tester) async {
+    final Key popupButtonKey = UniqueKey();
+    final Key popupButtonApp = UniqueKey();
+    final Key popupItemKey = UniqueKey();
+
+    await tester.pumpWidget(MaterialApp(
+      key: popupButtonApp,
+      home: Material(
+        child: Column(
+          children: <Widget>[
+            PopupMenuTheme(
+              color: Colors.pink,
+              shape: BeveledRectangleBorder(borderRadius: BorderRadius.circular(10)),
+              elevation: 6.0,
+              textStyle: const TextStyle(color: Color(0xfffff000), textBaseline: TextBaseline.alphabetic),
+              child: PopupMenuButton<void>(
+                key: popupButtonKey,
+                itemBuilder: (BuildContext context) {
+                  List<PopupMenuEntry<Object>> list = <PopupMenuEntry<Object>>[
+                    PopupMenuItem<void>(
+                      key: popupItemKey,
+                      child: const Text(''),
+                    ),
+                  ];
+                  return list;
+                },
+              ),
+            ),
+          ],
+        ),
+      ),
+    ));
+
+    await tester.tap(find.byKey(popupButtonKey));
+    await tester.pumpAndSettle();
+
+    final Material button = tester.widget<Material>(
+      find.descendant(
+        of: find.byKey(popupButtonApp),
+        matching: find.byType(Material),
+      ).last,
+    );
+    expect(button.color, Colors.pink);
+    expect(button.shape,
+        BeveledRectangleBorder(borderRadius: BorderRadius.circular(10)));
+    expect(button.elevation, 6.0);
+
+    final AnimatedDefaultTextStyle text = tester.widget<AnimatedDefaultTextStyle>(
+      find.descendant(
+        of: find.byKey(popupItemKey),
+        matching: find.byType(AnimatedDefaultTextStyle),
+      ).last,
+    );
+    expect(text.style.color, const Color(0xfffff000));
+  });
+}
+
+PopupMenuThemeData _popupMenuTheme() {
```

Comment:
It might be better to put this at the top of the file. I believe most test files in the framework follow that convention

Reply:
Done (I was looking at the [Bottom Sheet Theme](https://github.com/flutter/flutter/pull/31318/files#diff-10cf511a910f6f022c93559b114c640f) which seems to have this at the bottom of the test file?)

Reply:
I see, it must be a preference thing. The files I've worked on had them at the top of the file. Sorry about that!